### PR TITLE
Transfer - lists labels - English Translation

### PR DESCRIPTION
**Pull request by kvetoslavnovak**

### CODE REVIEW

**Comment by benjycui** [comment]
It seems that it is a but that `Transfer[titles]` is not under the control of `LocaleProvider`

**Comment by kvetoslavnovak** [comment]
@benjycui  should it be add as an independent new issue  or will you solve it  within this translation PR?

**Comment by benjycui** [comment]
I will fix this bug, then merge this PR.

Keep this PR open to remind me.

File: components/transfer/index.en-US.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To transfer the elements between two columns in an intuitive and efficient way.
 | onSelectChange | A callback function which is executed when selected items are changed. | Function(sourceSelectedKeys, targetSelectedKeys) | |
 | listStyle | A custom CSS style used for rendering the transfer columns. | Object |  |
 | className | A custom CSS class. | String |  |
-| titles | A set of titles that are sorted from left to right. | Array | ['源列表', '目的列表'] |
+| titles | A set of titles that are sorted from left to right. | Array | ['Source list', 'Target list'] |
 | operations | A set of operations that are sorted form top to bottom. | Array | [] |
 | showSearch | If included, a search box is shown on each column. | Boolean | false |
 | searchPlaceholder | The hint text of the search box. | String | 'Please input the content' |
```
Comment (visvadw) [inline components/transfer/index.en-US.md]: ` | searchPlaceholder | The hint text of the search box. | String | 'Search ...' |`

File: components/transfer/index.en-US.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ To transfer the elements between two columns in an intuitive and efficient way.
 | onSelectChange | A callback function which is executed when selected items are changed. | Function(sourceSelectedKeys, targetSelectedKeys) | |
 | listStyle | A custom CSS style used for rendering the transfer columns. | Object |  |
 | className | A custom CSS class. | String |  |
-| titles | A set of titles that are sorted from left to right. | Array | ['源列表', '目的列表'] |
+| titles | A set of titles that are sorted from left to right. | Array | ['Source list', 'Target list'] |
 | operations | A set of operations that are sorted form top to bottom. | Array | [] |
```
Comment (visvadw) [inline components/transfer/index.en-US.md]: ` | operations | A set of operations that are sorted from top to bottom. | Array | [] |`

**Comment by kvetoslavnovak** [comment]
@benjycui  I tried to track it. 
There is no property Transfer[titles] in LocaleProvider  https://github.com/ant-design/ant-design/blob/master/components/locale-provider/en_US.tsx (lines 29-34)
Trasnfer component has hardcoded titles as static defaultProps array https://github.com/ant-design/ant-design/blob/master/components/transfer/index.tsx (line 51)

File: components/transfer/index.en-US.md (hunk 1 above)
Comment (benjycui) [inline components/transfer/index.en-US.md]: should be `'Search here'`

File: components/transfer/index.en-US.md (hunk 2 above)
Comment (benjycui) [inline components/transfer/index.en-US.md]: Yep.. typo

**Comment by benjycui** [comment]
@kvetoslavnovak could you update your PR according to review?

**Comment by benjycui** [comment]
And this PR will be merged after https://github.com/ant-design/ant-design/pull/3318

**Comment by kvetoslavnovak** [comment]
Closing, will be merged after #3318

File: components/transfer/index.en-US.md (hunk 2 above)
Comment (kvetoslavnovak) [inline components/transfer/index.en-US.md]: Updated

File: components/transfer/index.en-US.md (hunk 1 above)
Comment (kvetoslavnovak) [inline components/transfer/index.en-US.md]: Updated with 'Search here'
(Maybe 'Search ...'  is better?)

**Pull request merged**